docs/source/concepts/FittingMinimizers.rst

```diff
@@ -0,0 +1,205 @@
+algorithm is a modified Gauss-Newton that introduces an adaptive term
+to prevent unstability when the approximated Hessian is not positive
+defined. An in-depth description of the methods is beyond the scope of
+these pages. More information can be found from the links and general
+references on optimization methods such as [Kelley1999]_ and
+[NocedalAndWright2006]_.
+
+Finally, :ref:`FABADA <FABADA>` is an algorithm for Bayesian data
+analysis. It is excluded from the comparison described below, as it is
+a substantially different algorithm.
+
+In most cases, the implementation of these algorithms is based on the
+`GNU Scientific Libraty routines for least-squares fitting
+<https://www.gnu.org/software/gsl/manual/html_node/Least_002dSquares-Fitting.html#Least_002dSquares-Fitting>`__
+
+**TODO: Exceptions that are not GSL methods? Levenberg-MarquardMD?**.
+
+
+Comparison of relative goodness of fit and run time
+---------------------------------------------------
+
+Here we describe a comparison of minimizers available in Mantid, in
+terms of how they perform when fitting several benchmark
+problems. This is a relative comparison in the sense that for every
+problem the best possible results with Mantid minimizers are given a
+top score of "1". The ranking is continuous and the score of a
+minimizer represents the ratio between its performance and the
+performance of the best. We compare accuracy and run time.
+
+.. Q: accuracy metric: sum of squared residuals or chi2 variants
+
+For example, a ranking of 1.25 for a minimizer for a given problem
+means:
+
+- Referring to the accuracy of a minimizer, it produces a solution
+  with squared residuals 25% larger than the best solution in Mantid.
+- Referring to the run time, it takes 25% more time than the fastest
+  minimizer.
+
+All the minimizers available in Mantid 3.7 were compared, with the
+exception of FABADA which belongs to a different class of methods and
+would not be compared in a fair manner. Minimizers are used here as
+"black boxes", without any special initialization, constraints, or
+rewriting of the equations to fit.
+
+Accuracy is measured using the sum of squared fitting errors as
+metric, or "ChiSquared" as defined in :ref:`Fit
+<algm-CalculateChiSquared>`, where the fitting errors are the
+difference between the expected outputs and the outputs calculated by
+the model fitted: :math:`\chi_{1}^{2} = \sum_{i} (y_i - f_i)^2` (see
+:ref:`CalculateChiSquared <algm-CalculateChiSquared>` for full details
+and different variants).  Run time is measured for the execution of
+the :ref:`Fit <algm-Fit>` algorithm for an equation and dataset
+previously created.
+
+.. There would be two alternative for the errors:
+   1. Without errors, as it is
+   2. With "simulated" sqrt(X) errors
+   Here we use 2 because Levenberg-Marquardt doesn't work with 1 ('Unweighted least squares' cost function
+
+The cost function used in all cases is `Least squares` (weighted by
```

Comment:
Note: clarify here that "Least squares" is the name in Mantid, and that this is the default choice of cost function. It is the weighted version.
The name of the unweighted version is "Unweighted least squares".